README.md

```diff
@@ -6,12 +6,12 @@
 # flake.nix
 inputs = {
   home-manager = {
-    url = "github:nix-community/home-manager/release-24.05";
+    url = "github:nix-community/home-manager/release-24.11";
     inputs.nixpkgs.follows = "nixpkgs";
   };
-  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.11";
   nixvim = {
-    url = "github:nix-community/nixvim/nixos-24.05";
+    url = "github:nix-community/nixvim/nixos-24.11";
```

Comment:
Shouldn't these be all pointing to unstable? Ci is testing stable using override inputs.